code/modules/mining/ore.dm

```diff
@@ -121,4 +121,12 @@
 		var/obj/item/device/core_sampler/C = W
 		C.sample_item(src, user)
 	else
-		return ..()
\ No newline at end of file
+		return ..()
+
+/obj/item/weapon/ore/attack(mob/living/M as mob, mob/living/user as mob)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.species.eat_ore == 1)
+			H.eat_ore(src)
+			return
+	..()
```

Comment:
```suggestion
//VOREStation Add
/obj/item/weapon/ore/attack(mob/living/M as mob, mob/living/user as mob)
	if(ishuman(M))
		var/mob/living/carbon/human/H = M
		if(H.species.eat_ore == 1)
			H.eat_ore(src)
			return
	..()
//VOREStation Add End
```

Needs vorestation edit comments. You can use the commit button on this suggestion to apply it.

Reply:
Cheers, good catch

This new suggestions feature rocks.